ARXIVNG-1786 add messaging for double-blind condition

eawoods opened the pull request:
Erick: to review for code, design, and language
Jim: to review for design and language

Scope: this adjustment fulfills a tricky ticket, "it must be clear to submitter that submissions cannot be anonymous" - addressing a common misconception brought up by Jake that a withdrawal will remove a submission from public view.

I decided to put this up without trying to wordsmith it to death, would rather have input from the admin team on that. A paired help page might be a great idea, so that we can have a small amount of text and a link in the UI with more explanation elsewhere. I'd rather have even less text (and a link) if this concept can be expressed in a better, shorter way.

## Screenshot
![Screen Shot 2019-04-01 at 10 34 49 AM](https://user-images.githubusercontent.com/17456668/55336425-e19dea00-546a-11e9-8a5d-d2bd6856c15a.png)

jimentwood commented:
For Verify User step
maybe something like: "The name and email address of the submitter will be viewable to registered arXiv users." With link to: https://arxiv.org/help/email-protection

Then on the Authorship step
something like: "Complete and accurate authorship is required and will be displayed in the public metadata". 
link to https://arxiv.org/help/prep#author
We can beef up the help page text - https://arxiv-org.atlassian.net/browse/ARXIVOPS-682

eawoods commented:
Update: added text per Jim's excellent suggestions. I changed the color of the authorship screen to blue, since it seemed more informatic than warning...but maybe both should be blue. We've been using the orange one gently as a "more important, might cause problems" emphasizer of sorts. I'm on the fence about the colors for these particular tidbits.

![Screen Shot 2019-04-01 at 4 31 27 PM](https://user-images.githubusercontent.com/17456668/55358404-512ccd00-549d-11e9-9111-23dfb42d8e37.png)

![Screen Shot 2019-04-01 at 4 41 38 PM](https://user-images.githubusercontent.com/17456668/55358411-54c05400-549d-11e9-8d8c-e4c3236fc962.png)
